src/account/default.ts

```diff
@@ -327,6 +327,30 @@ export class Account extends Provider implements AccountInterface {
     );
   }
 
+  /**
+   * First check if contract is already declared, if not declare it
+   * If contract already declared returned transaction_hash is ''.
+   * Method will pass even if contract is already declared
+   * @param payload DeclareContractPayload
+   * @param transactionsDetail (optional) InvocationsDetails = {}
+   * @returns DeclareContractResponse
+   */
+  public async declareIfNot(
```

Comment:
should we add tests for this method itself?

Reply:
a18b26d8fa45cae4e6d3516f794916ca1e8f50e1